engine/src/main/scala/cromwell/jobstore/SqlJobStore.scala

```diff
@@ -56,6 +56,7 @@ class SqlJobStore(sqlDatabase: JobStoreSqlDatabase) extends JobStore {
       case "Int" => WdlIntegerType
       case "Float" => WdlFloatType
       case "Boolean" => WdlBooleanType
+      case "File" => WdlFileType
```

Comment:
Actually upon further reflection I'm going to ticket this and revert these changes.  This change is unrelated to migrations but rather to a bug @ruchim discovered in the call cache result code.  I realized I had the same bug in the job store code and was slipping the changes into this PR, but given the discussion here I'm going to back away from that.